lib/jekyll/renderer.rb

```diff
@@ -2,20 +2,41 @@
 
 module Jekyll
   class Renderer
-    attr_reader :document, :site, :payload
+    attr_reader :document, :site
+    attr_writer :layouts, :payload
 
-    def initialize(site, document, site_payload = nil)
+    def initialize(site, document, site_payload = nil, layouts: nil)
```

Comment:
@envygeeks Are named parameters not allowed in jruby? https://travis-ci.org/jekyll/jekyll/jobs/156357192